jbi/router.py

```diff
@@ -87,7 +90,7 @@ def get_whiteboard_tags(
 @router.get("/jira_projects/")
 def get_jira_projects():
     """API for viewing projects that are currently accessible by API"""
-    visible_projects: list[dict] = jira_visible_projects()
+    visible_projects: list[dict] = jira.fetch_visible_projects()
```

Comment:
Do you think we should have our own Jira class with such methods? Or can we a general guideline on when to use a module function and when to use a client method?

Reply:
> Or can we establish a general guideline on when to use a module function and when to use a client method

I like this approach. I was thinking about this a bit. I think the separation of responsibilities should be [^1]:
- client class: calls the REST API, returns JSON
- module functions or a service class: uses the client to fetch data, then processes it

If an action wants to just fetch `json`, it could do that with the REST client. But I'd imagine they'd more than likely use a module function or service class method.

> Do you think we should have our own Jira class with such methods?

For now, I don't think the service module functions share any state that would make combining them into a class useful. But if that starts to become the case, maybe a service class makes sense.

[^1]: I know this isn't how the Bugzilla module is at the moment, but maybe we could make that change?